pkg/skaffold/deploy/helm/helm_test.go

```diff
@@ -693,13 +701,30 @@ func TestHelmDeploy(t *testing.T) {
 			helm: testDeployUpgradeOnChange,
 		},
 		{
-			description: "deploy error remote chart without skipBuildDependencies",
+			description: "deploy remote chart",
+			commands: testutil.
+				CmdRunWithOutput("helm version --client", version31).
+				AndRunErr("helm --kube-context kubecontext get all skaffold-helm-remote --kubeconfig kubeconfig", fmt.Errorf("Error: release: not found")).
+				AndRun("helm --kube-context kubecontext install skaffold-helm-remote stable/chartmuseum --repo https://charts.helm.sh/stable --kubeconfig kubeconfig").
+				AndRun("helm --kube-context kubecontext get all skaffold-helm-remote --template {{.Release.Manifest}} --kubeconfig kubeconfig"),
+			helm: testDeployRemoteChart,
+		},
+		{
+			description: "deploy remote chart with version",
+			commands: testutil.
+				CmdRunWithOutput("helm version --client", version31).
```

Comment:
nit: does helm require `semver` versions?  I don't think so from:
https://helm.sh/docs/helm/helm_dependency/#synopsis

Reply:
We require certain versions due to functionality being introduced.  In this case, I just copied from the previous test.